docs/car-park-skills.md

```diff
@@ -83,10 +83,23 @@ aea generate-key fetchai
 aea add-key fetchai fet_private_key.txt
 ```
 
-And, fund the carpark client agent:
+To fund the carpark client agent on test-net you need to request some test tokens from a faucet.
+
+First, print the address:
 ``` bash
-cd ..
-python scripts/fetchai_wealth_generation.py --private-key car_data_buyer/fet_private_key.txt --amount 10000000000 --addr alpha.fetch-ai.com --port 80
+aea get-address fetchai
+```
+
+This will print the address to the console. Copy the address into the clipboard and request test tokens from the faucet [here](https://explore-testnet.fetch.ai/tokentap). It will take a while for the tokens to become available.
+
+Second, after some time, check the wealth associated with the address:
+``` bash
+aea get-wealth fetchai
```

Comment:
if I don't follow the step above (i.e. request of test tokens) I get 
```error: Unsupported ledger identifier.```

Reply:
sorry, I forgot the step above (i.e. populating `ledger_apis`).

However, after adding:
```yaml
ledger_apis:
  fetchai:
    address: testnet.fetch-ai.com
    port: 80
```

I get:
```error: Either network or addr and port must be specified.```

Reply:
Will fix

Reply:
ok `"network": "testnet"` worked

Reply:
is there a way to make it work with the pair `addr` and `port` at the moment?

Reply:
No I couldn't get that to work and I looked at the available hosts and there seems to be some issues with them being resolved correctly